.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: go
+
+go:
+  - 1.7
+  - 1.8
+  - tip
+
+sudo: required
```

Comment:
this line sucks..

https://docs.travis-ci.com/user/ci-environment/

If `sudo` isn't set, then Travis will use the container infrastructure. That means that build starts very fast on a running VM. Otherwise, travis will schedule the creation of a new VM for each build... making the CI feedback really slow.

This is not critical, but helps a lot..

To drop this, we only need to install the required tools (awscli, azure, jq, etc) in a bin directory somewhere else and set PATH accordingly.